package.json

```diff
@@ -42,13 +42,12 @@
     "@babel/preset-react": "^7.16.0",
     "@types/nprogress": "^0.2.0",
     "babel-preset-minify": "^0.5.1",
-    "eslint": "^8.1.0",
+    "eslint": "^8.2.0",
     "eslint-config-prettier": "^8.3.0",
     "husky": "^7.0.4",
     "lint-staged": "^11.2.6",
-    "next": "^12.0.2",
-    "prettier": "^2.4.1",
-    "react": "^17.0.2"
```

Comment:
How is it working without react? Are you able to build it properly? `npm run build`?

Reply:
First, I removed react because of an error related to functional component error. Then I saw it can run without react. That's why I removed it. I tested it again by deleting previous node-modules and reinstalling it just to make sure it's working fine. 
![image](https://user-images.githubusercontent.com/74945038/140653686-8e7af3e8-4234-450b-8e24-4f4cf47b48d6.png)

Even I just tested this out with github action just to make sure it wasn't because of cache 
![image](https://user-images.githubusercontent.com/74945038/140653919-6b05ae5d-8a39-4485-979e-ec08ca50a5f1.png)

Reply:
It won't compile if we remove next.

Reply:
I'll readd it when rewritting in ts.